test/models/mongo/CisCYAModelSpec.scala

```diff
@@ -84,6 +84,35 @@ class CisCYAModelSpec extends UnitTest
     }
   }
 
+  ".isPeriodDataUpdatedFor" should {
```

Comment:
This is renamed. Can you change to: .periodDataUpdated

Reply:
updated